clients/java/src/test/java/com/phrase/client/api/UploadsApiTest.java

```diff
@@ -106,7 +107,7 @@ public void uploadCreateTest() throws ApiException, IOException, InterruptedExce
         Boolean autotranslate = null;
         Boolean markReviewed = null;
         Boolean tagOnlyAffectedKeys = null;
-        Upload response = api.uploadCreate(projectId, xPhraseAppOTP, branch, file, fileFormat, localeId, tags, updateTranslations, updateDescriptions, convertEmoji, skipUploadTags, skipUnverification, fileEncoding, localeMapping, formatOptions, autotranslate, markReviewed, tagOnlyAffectedKeys);
+        Upload response = api.uploadCreate(projectId, xPhraseAppOTP, branch, file, fileFormat, localeId, tags, updateTranslations, updateTranslationKeys, updateDescriptions, convertEmoji, skipUploadTags, skipUnverification, fileEncoding, localeMapping, formatOptions, autotranslate, markReviewed, tagOnlyAffectedKeys);
```

Comment:
it's interesting that this was needed. I thought this parameter is not required? any idea why it's part of the signature?

Reply:
I checked the newly built Java client, in fact other params that are marked as optional are all passed to the method. I guess all these arguments have to be passed to the method, we can set `null` for the ones that are optional.

Reply:
Because if I am not wrong, a Java method would expect all the options to be passed, its within the method where we check if the option is present, then assign it to the payload.
![Screenshot 2024-04-17 at 15 34 55](https://github.com/phrase/openapi/assets/141136168/9752f83f-da48-4cd4-9c99-61bff0f82004)

Reply:
interesting, technically that means that any such change should be marked as breaking for java lib :( but I guess we won't be doing that.

Reply:
I don't think so. This change shouldn't be a breaking one, we added an optional field. However, seems to me that Java generator is not correct so it generates a breaking change for Java. I don't think we have a way right now to declare breaking changes only for a single platform.
So, I'd keep this a minor version bump as usual, but we should be aware of this Java generator quirk, and perhaps try to address it in the future (perhaps updating the Java templates to newer version would be enough, although even that is no easy job).